src/Entity/Reviewer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace DemoCQRSHooksUsage\Entity;
+
+use PrestaShop\PrestaShop\Adapter\Entity\ObjectModel;
+
+class Reviewer extends ObjectModel
```

Comment:
Not for today, but we could add a `@todo` for a 2nd version: handle this data using Doctrine instead of ObjectModel 😄

Reply:
Created as an improvement suggestion https://github.com/friends-of-prestashop/demo-cqrs-hooks-usage-module/issues/1